spiped/main.c

```diff
@@ -27,6 +28,12 @@ usage(void)
 	exit(1);
 }
 
+static void
+signal_handler(int signo)
+{
+	exit(0);
```

Comment:
Unused variables need to be explicitly designated as such:

        (void)signo; /* UNUSED */

In addition to making the programmer intention clear, it prevents compiler (and static analysis tool) warnings.